### ISSUE TITLE
problem with W3C validation

### ISSUE THREAD

**Issue by Jimmi08** [e107inc/e107#3585]
I am getting warnings with W3C validation with new theme.

![image](https://user-images.githubusercontent.com/5429548/50290627-8d976880-046c-11e9-9152-c1d2d5aa99b0.png)

I don't know if this is important or not, but I wanted to note this.

**Comment by tgtje** [comment]
Just my 2 cents : it is when html5 (check) as type= is no longer needed , so a warning, not an error.
So type=text/css... and type=text/java..... could be discarded/not used (for html5).
Most likely a validation using 4.0.1 Trans.. should show no warning.
( https://validator.w3.org/#validate_by_uri+with_options ) just change Doc..type select..

**Comment by Jimmi08** [comment]
@tgtje yes, you are right. But site has html5 doctype, not 4.0.1, so there should be way how to set this and then output of e107::css would be different.

**Comment by Moc** [comment]
> I don't know if this is important or not, but I wanted to note this.

Not important as it does not have an impact on functionality but seeing we're using HTML5 as default doctype mostly nowadays, it would be something to change in core for sure. We just need to think of backwards compatibility here with themes using other doctypes. Not sure what the best option is.
